index.js

```diff
@@ -18,25 +16,38 @@ Fs.readdirSync(Path.join(__dirname, 'helpers')).forEach(function (file) {
 
 /**
  * Theme renderer constructor
+ * @param {Object} settings
+ * @param {Object} themeSettings
  * @param {Object} assembler
  */
-module.exports = function (assembler) {
+module.exports = function (settings, themeSettings, assembler) {
     var self = this;
 
     self.handlebars = Handlebars.create();
 
     self.handlebars.templates = {};
 
-    self.options = internals.options;
+    self.settings = settings || {};
+    self.themeSettings = themeSettings || {};
+
     self.translate = null;
     self.inject = {};
     self.helpers = [];
     self.decorators = [];
 
     _.each(Helpers, function (Helper) {
-        self.helpers.push(new Helper(self.handlebars));
+        new Helper(self.handlebars).register(self);
```

Comment:
I wonder if the instantiation could be run once at the start of the application, and then just registered with an instance of paper every time a request is made, rather than calling `new` every request.

Basically just removing the `new` combined constructor + register approach, and having something like this instead:

```
_.each(helpers, function (helper) {
helper.register(self);
});
```